Issue610 cdl extra basic blocks

JayHuLBL opened the pull request:
I will add other blocks on a new branch

mwetter commented on Buildings/Experimental/OpenBuildingControl/CDL/Constants/package.mo:
```diff
@@ -0,0 +1,133 @@
+within Buildings.Experimental.OpenBuildingControl.CDL;
```
The file CDL.Constants should be deleted.

mwetter commented on Buildings/Experimental/OpenBuildingControl/CDL/Continuous/Derivative.mo:
```diff
@@ -0,0 +1,115 @@
+<p>
+If you would like to be able to change easily between different
+transfer functions (FirstOrder, SecondOrder, ... ) by changing
+parameters, use the general block <b>TransferFunction</b> instead
```
use <code> instead of <b>

mwetter commented on Buildings/Experimental/OpenBuildingControl/CDL/Continuous/Derivative.mo:
```diff
@@ -0,0 +1,115 @@
+transfer functions (FirstOrder, SecondOrder, ... ) by changing
+parameters, use the general block <b>TransferFunction</b> instead
+and model a derivative block with parameters<br/>
+b = {k,0}, a = {T, 1}.
```
use <code>

mwetter commented on Buildings/Experimental/OpenBuildingControl/CDL/Sources/Pulse.mo:
```diff
@@ -0,0 +1,170 @@
+  parameter Real offset=0 "Offset of output signals";
+  parameter Modelica.SIunits.Time startTime=0
+    "Output = offset for time < startTime";
+  Interfaces.RealOutput y annotation (Placement(
```
comment is missing

mwetter commented on Buildings/Experimental/OpenBuildingControl/CDL/Sources/Pulse.mo:
```diff
@@ -0,0 +1,170 @@
+        transformation(extent={{100,-10},{120,10}})));
+
+protected
+  Modelica.SIunits.Time T_width=period*width/100;
```
comment is missing

JayHuLBL commented:
@mwetter I have revised the classes following your comments: 
[x] added necessary comments 
[x] used <code></code>
[x] added table in `Buildings.Experimental.OpenBuildingControl.CDL.Types.Init`
[x] updated block icons

mwetter commented:
@JayHuLBL Thanks. I merged it and will post to the website before the meeting. Feel free to commit to `issue610_cdl_extra_basic_blocks` again if you have more changes.
